tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -243,14 +243,60 @@ public function userUpdatesTheLastShareWithFollowingUsingGraphApi($user, TableNo
 		$response = $this->featureContext->shareNgGetLastCreatedUserGroupShare();
 		$permissionID = json_decode($response->getBody()->getContents())->value[0]->id;
 		$this->featureContext->setResponse(
-			$this->updateShare(
+			$this->updateLinkShare(
 				$user,
 				$table,
 				$permissionID
 			)
 		);
 	}
 
+	/**
+	 * @param string $user
+	 * @param TableNode $body
+	 * @param string $permissionID
+	 *
+	 * @return ResponseInterface
+	 */
+	public function updateLinkShare(string $user, TableNode  $body, string $permissionID): ResponseInterface {
```

Comment:
shouldn't this be `updateShare` and the later one `updateLinkShare`?
```suggestion
	public function updateShare(string $user, TableNode  $body, string $permissionID): ResponseInterface {
```